src/scss/components/_song-list.scss

```diff
@@ -103,18 +103,19 @@
 }
 
 .songlist__table {
-  border: none;
   background-color: $color-white;
+  border: none;
+  border-collapse: collapse;
   margin: 0 rhythm(-2);
   padding: 0;
   position: relative;
 
   @include breakpoint($until: medium) {
     display: block;
+    width: 109%;
```

Comment:
> I think I fixed the issue with the song list table shortening for the shorter band names by adjusting the width to 109%. I just eyeball-ed it with the emulator to see where it would line up with the edge but not go over too far. Feels a little hacky, maybe there's a better solution?

It looks like `100%` makes it the width of its container, without taking the negative side margins (that make it stretch to the full device width) into account. The negative margins are `2 * 16px`, so rather than the ol eyeballed percentage, a more precise way to accomplish what you're trying to do would be `width: calc(100% + 32px);`.

(`calc` is handy and worth reading up on if you haven't already.)

Reply:
ooooh that's super dope. Good to know about. Thanks!